.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve the ohsome API.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Bug Description
+A clear and concise description of what the bug is.
+
+### General Information
+ - Affected endpoint(s) [e.g. /contributions/count]
+ - Affected parameter(s) [e.g. bpolys, format]
+ - HTTP method used [e.g. POST]
+ - Used tool/library for the request [e.g. cURL, Postman]
+
+### How to Reproduce
+Steps [e.g.] to reproduce the behaviour:
+1. Start locally the ohsome API
+2. Send this request [URL] (http://localhost:8080/contributions/count?bboxes=8.67%2C49.39%2C8.71%2C49.42&time=2017-01-01%2F2017-01-02%2FP1Y)
+3. See error message 'Unable to find...'
+
+### Expected Behaviour
+A clear and concise description of what you expected to happen [e.g. "The error message should be..."]
+
+### Further Information
+
+#### Error messages, logs, screenshots
+If applicable, add printed error messages, log files or screenshots to help explain your problem.
+
+#### Additional information
```

Comment:
```suggestion
#### Additional Information
```

Reply:
Since it is a subparagraph I would use capitalization only for the first word.

Reply:
Why the distinction between "normal" headings and headings of subsections? I would recommend to always use one single consistent style for headings throughout a text.